### PR TITLE
Update phones-displays-deploy.md

### PR DESCRIPTION
**Pull request by kruthikaponnusamy**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/devices/phones-displays-deploy.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Deploy Teams phones and Teams displays using Intune"
+title: "Deploy Teams phones, Teams displays and Microsoft Teams Rooms on Android using Intune"
```
Comment (JohanFreelancer9) [inline Teams/devices/phones-displays-deploy.md:2]:
```suggestion
title: "Deploy Teams phones, Teams displays, and Microsoft Teams Rooms on Android using Intune"
```

File: Teams/devices/phones-displays-deploy.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ localization_priority: Normal
 description: "This article provides an overview of and features supported by Microsoft Teams displays."
 ---
 
-# Deploy Teams phones and Teams displays using Intune
+# Deploy Teams phones, Teams displays and Microsoft Teams Rooms on Android using Intune
```
Comment (JohanFreelancer9) [inline Teams/devices/phones-displays-deploy.md:21]:
```suggestion
# Deploy Teams phones, Teams displays, and Microsoft Teams Rooms on Android using Intune
```

File: Teams/devices/phones-displays-deploy.md [hunk 3]
```diff
@@ -18,13 +18,13 @@ localization_priority: Normal
...
 
-This article gives you an overview of how to deploy. Teams phones and Teams display using Intune.
+This article gives you an overview of how to deploy. Teams phones, Teams displays and Microsoft Teams Rooms on Android using Intune.
```
Comment (JohanFreelancer9) [inline Teams/devices/phones-displays-deploy.md:23]:
```suggestion
This article gives you an overview of how to deploy Teams phones, Teams displays, and Microsoft Teams Rooms on Android using Intune.
```

File: Teams/devices/phones-displays-deploy.md [hunk 4]
```diff
@@ -18,13 +18,13 @@ localization_priority: Normal
...
+Conditional Access is an Azure Active Directory (Azure AD) feature that helps you to ensure devices accessing your Office 365 resources are properly managed and are secure.  If you apply Conditional Access policies to the Teams service, Android devices (including Teams phones, Teams displays and Microsoft Teams Rooms on Android) that access Teams needs to be enrolled into Intune and their settings need to comply with your policies.  If the device isn't enrolled into Intune, or if it's enrolled but its settings don't comply with your policies, Conditional Access will prevent a user from signing in to or using the Teams app on the device.
```
Comment (JohanFreelancer9) [inline Teams/devices/phones-displays-deploy.md:27]:
```suggestion
Conditional Access is an Azure Active Directory (Azure AD) feature that helps you to ensure devices accessing your Office 365 resources are properly managed and are secure.  If you apply Conditional Access policies to the Teams service, Android devices (including Teams phones, Teams displays, and Microsoft Teams Rooms on Android) that access Teams need to be enrolled into Intune and their settings need to comply with your policies.  If the device isn't enrolled into Intune, or if it's enrolled but its settings don't comply with your policies, Conditional Access will prevent a user from signing in to or using the Teams app on the device.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kruthikaponnusamy, kindly commit the suggested changes. Many thanks.

**Comment by kruthikaponnusamy** [comment]
I committed the changes. Do I need to close the PR now?

**Review by JohanFreelancer9 — APPROVED** [review]
@kruthikaponnusamy - thank you for committing the changes. No, please don't close the PR. It needs to be approved by the doc owner/manager and will then be merged.

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**